modules/hadoop/src/main/tut/README.md

```diff
@@ -0,0 +1,48 @@
+# Hadoop module for PureConfig
+
+Adds support for selected [Hadoop](http://hadoop.apache.org/) classes to PureConfig.
+
+## Add pureconfig-hadoop to your project
+
+In addition to [core PureConfig](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-hadoop" % "0.9.0"
+```
+
+Also, `pureconfig-hadoop` depends on `hadoop-common` with `provided` scope. It's mean that you should explicitly add a dependency on `hadoop-common` or any other Hadoop library who depends on `hadoop-common`. Usually it would be something like this:
```

Comment:
The wording here is a little odd. I would suggest something like "This means that ..." and "... library which depends ...".